src/main/java/com/massivecraft/factions/Conf.java

```diff
@@ -192,12 +192,14 @@
     public static boolean territoryDenyEndermanBlocksWhenOffline = true;
     public static boolean safeZoneDenyBuild = true;
     public static boolean safeZoneDenyUseage = true;
+    public static Set<Material> safezoneAllowUseageMaterials = new HashSet<>();
     public static boolean safeZoneBlockTNT = true;
     public static boolean safeZonePreventAllDamageToPlayers = false;
     public static boolean safeZoneDenyEndermanBlocks = true;
     public static boolean safeZoneTerritoryDisablePVP = true;
     public static boolean warZoneDenyBuild = true;
     public static boolean warZoneDenyUseage = true;
+    public static Set<Material> warzoneAllowUseageMaterials = new HashSet<>();
```

Comment:
We wanna null check this, as GSON makes empty sets null, so if a uses makes the set empty and we load from a file it'll be null. Not a major issue, but better safe then sorry.

Reply:
Same for the safezone set.